tests/external/statefulset_keycloak_test.go

```diff
@@ -9,8 +9,16 @@ import (
 	"github.com/projectsyn/component-keycloak/common"
 )
 
+var (
+	keycloakStsPath = testPath+"/01_keycloak_helmchart/keycloak/templates"
+)
+
+func Test_Keycloak_StatefulSet_ApiVersion(t *testing.T) {
+	common.StatefulSetApiVersionTest(t, keycloakStsPath, "keycloak")
+}
+
 func Test_Keycloak_StatefulSet_Secrets(t *testing.T) {
-	subject := common.DecodeStatefulsetV1(t, testPath+"/01_keycloak_helmchart/keycloak/templates/statefulset.yaml")
+	subject := common.DecodeStatefulsetV1(t, keycloakStsPath+"/statefulset.yaml")
```

Comment:
Why do we specifically test the Api version if the other tests below would already fail the decoding if the version is wrong?